overlord/configstate/configcore/netplan.go

```diff
@@ -196,11 +196,16 @@ func handleNetplanConfiguration(tr config.Conf, opts *fsOnlyContext) error {
 		}
 	}
 
+	// re-try reaching the store to guard against flaky networks
 	var storeReachableBefore bool
-	if err := storeReachable(tr.State()); err == nil {
-		storeReachableBefore = true
+	for i := 0; i < 5; i++ {
+		if err := storeReachable(tr.State()); err == nil {
+			storeReachableBefore = true
+			break
+		}
+		logger.Debugf("store reachable before netplan %v (retry %v)", storeReachableBefore, i)
```

Comment:
this will always print false no? we should log also outside?  maybe this loops should become a little helper? that takes "before" or "after" ?